apps/site/package.json

```diff
@@ -107,6 +114,9 @@
     "typescript": "~5.8.2",
     "typescript-eslint": "~8.31.1",
     "unified": "^11.0.5",
-    "user-agent-data-types": "0.4.2"
+    "user-agent-data-types": "0.4.2",
+    "@opennextjs/cloudflare": "^1.0.0-beta.3",
+    "wrangler": "^4.13.0",
+    "@flarelabs-net/wrangler-build-time-fs-assets-polyfilling": "^0.0.0"
```

Comment:
Can you alphabetize this?

Reply:
sorry I didn't notice the ordering 🙇 , done 🙂